spec/controllers/contact_controller_spec.rb

```diff
@@ -48,8 +50,7 @@
   end
   context 'with contact e-mail set' do
     before do
-      AppConfig.first
-        .update_attributes(contact_link_location: 'contact@example.com')
+      allow(@ac).to receive(:contact_link_location) { 'contact@example.com' }
```

Comment:
Perfect; I think in the interest of keeping this as a useful learning tool (and for tests to also serve as some kind of documentation) the explicit syntax is worth the extra lines :stuck_out_tongue:.